docs/en/remote-flow.rst

```diff
@@ -263,11 +280,34 @@ where a non-normative example in the form of decoded header and payload is shown
   }
   .
   {
-    "scope": "PersonIdentificationData WalletAttestation",
-    "client_id_scheme": "entity_id",
     "client_id": "https://relying-party.example.org",
     "response_mode": "direct_post.jwt",
     "response_type": "vp_token",
+    "dcql_query": {
```

Comment:
Normative details of the DCQL query are missing

Reply:
here: https://github.com/italia/eudi-wallet-it-docs/pull/507/files#diff-49b422a54bf6c8ac0ba7c115989fc1e59cca9b661758d9df668df4ee8bd0d406R347